doc/apiary/iotagent.apib

```diff
@@ -87,7 +87,7 @@ Fields in JSON object representing a configuration group are:
 |`attributes`            |Mapping for protocol parameters to entity attributes.<br> `object_id` (string, mandatory): protocol parameter to be mapped.<br> `name` (string, mandatory): attribute name to publish.<br> `type`: (string, mandatory): attribute type to publish.|
 |`static_attributes`     |Attributes published as defined.<br> `name` (string, mandatory): attribute name to publish.<br> `type` (string, mandatory): attribute type to publish.<br> `value` (string, mandatory): attribute value to publish.|
 | `timestamp`.           |(optional, boolean): This field indicates if an attribute 'TimeInstant' will be added (true) or not (false). If this field is omitted, the global IotAgent configuration timestamp will be used.|
-| `autoprovision`.       |(optional, boolean): If true, APPEND is used upon measure arrival (thus effectively allowing autoprovisioned devices). If false, UPDATE is used open measure arrival (thus effectively avoiding autoprovisioned devices). This field is optional, so if it omitted then the global IoTAgent appendModel configuration is used.|
+| `autoprovision`.       |(optional, boolean): If true, APPEND is used upon measure arrival (thus effectively allowing autoprovisioned devices). If false, UPDATE is used open measure arrival (thus effectively avoiding autoprovisioned devices), and device is even not able to be autoprovisioned. The default is `true`. This field is optional, so if it omitted then the global IoTAgent appendModel configuration is used.|
```

Comment:
It would be great add also a small section in advanced-topics.md document about autoprovision. @mapedraza will take care of it.

Reply:
Done in PR https://github.com/telefonicaid/iotagent-node-lib/pull/1068

NTC (in the present PR)